.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -1,17 +1,19 @@
 ---
 name: Documentation issue
 about: Issues related to documentation.
-title: '[Docs]'
-labels: 'docs'
-assignees: ''
+title: "[Docs]"
+labels: "docs"
+assignees: ""
```

Comment:
Please revert this change

Reply:
changes done. please check if anything wrong.